.github/workflows/release.yml

```
@@ -1,54 +1,54 @@
-        if: steps.release.outputs.released == 'true'
-        uses: python-semantic-release/publish-action@v10.5.3
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          tag: ${{ steps.release.outputs.tag }}
-
-      - name: 🐍 Publish to PyPI
-        if: steps.release.outputs.released == 'true'
-        env:
-          UV_PUBLISH_TOKEN: ${{ secrets.PYPI_TOKEN }}
-        run: uv publish
+name: Release & Publish
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)